shared/message/input/gazebo/Torso.proto

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2019 NUbots <nubots@nubots.net>
+ */
+syntax = "proto3";
+
+package message.input.gazebo;
+
+import "Matrix.proto";
+
+message Torso {
+    mat4 Hcw = 1;
```

Comment:
Did we want to add velocity to this message too? Or anything else that would be useful for measuring true torso movement?

Reply:
CoM (if we can get it) and velocity are probably good additions

Reply:
Cant immediately see how to get CoM, we can figure it out at some later point, along with intertia tensors